account-integrations/safe/src/SafeWebAuthnPlugin.sol

```diff
@@ -87,20 +88,22 @@ contract SafeWebAuthnPlugin is BaseAccount {
             uint i = 0;
             uint dataLen = 32;
             uint256 paramLen = abi.decode(userOp.signature[i:i+dataLen], (uint256));
-            // Fixed-length params (bytes1, (uint256?), uint256, uint256[2], uint256[2]). Expect 8 slots = 256 bytes
+            // Fixed-length params (bytes1, (uint256?), bytes32, uint256, uint256[2], uint256[2]). Expect 9 slots = 256 bytes
             i += dataLen; // advance index
 
             // decode fixed length params (values to memory)
-            dataLen = 3 * 32; //lenFixedParams - 32; // -32 already read length
+            dataLen = 4 * 32; //lenFixedParams - 32; // -32 already read length
             (
                 s.authenticatorDataFlagMask,
                 , // some number
+                s.clientChallenge,
                 s.clientChallengeDataOffset
             ) = abi.decode(
                 userOp.signature[i:i+dataLen],
                 (
                     bytes1,
                     uint256, //not sure what is encoded here
+                    bytes32,
```

Comment:
^I'm not sure why this is the order for the decoding, I originally assumed it would have been bytes before uints like so:
```solidity
(
    bytes1,
    bytes32,
    uint256, 
    bytes32
)
```

The screenshot below shows the difference between the two signatures. The older signature without the `clientChallenge` is on the left, and the newer signature with the `clientChallenge` is on the right:

![Screenshot 2023-09-15 at 16 12 30](https://github.com/getwax/wax/assets/54913924/08487286-6b19-4e93-8e07-01ef52bd6205)

You can see where the clientChallenge was added, look for `353a3ed5a0441919f1c639a46931de872ac3357de2ce5aa2d68c2639df54189d`. There were some other changes which I'm not too sure about. It looks like the mystery `uint256` changed. The value in the first 32 bytes was also changed, again not sure what that change means - @jzaki any ideas?